Externals/cpp-optparse/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+check_and_add_flag(CXX11 -std=c++11)
+
+set(SRCS OptionParser.cpp)
```

Comment:
Add the header too please. I know they're not added anywhere else but we've got to start somewhere!